atom/browser/atom_browser_main_parts.cc

```diff
@@ -20,28 +25,61 @@
 #include "atom/common/api/atom_bindings.h"
 #include "atom/common/asar/asar_util.h"
 #include "atom/common/node_bindings.h"
+#include "base/base_switches.h"
 #include "base/command_line.h"
-#include "base/threading/thread_task_runner_handle.h"
+#include "base/feature_list.h"
+#include "base/message_loop/message_loop.h"
+#include "base/path_service.h"
+#include "base/strings/string_number_conversions.h"
+#include "base/strings/utf_string_conversions.h"
+#include "brightray/browser/brightray_paths.h"
+#include "brightray/common/application_info.h"
```

Comment:
Can you move `brightray/browser/brightray_paths.h` in this PR, this file was the last user under `brightray`.

Reply:
@deepak1556 I was planning to do this in the last PR

Reply:
@deepak1556 https://github.com/electron/electron/pull/15240/commits/9c188cbc81fddab9866d943bb58f1518038af0bc